src/schoology-extractor/.env.example

```diff
@@ -0,0 +1,4 @@
+SCHOOLOGY_KEY=[YOUR_SCHOOLOGY_KEY]
+SCHOOLOGY_SECRET=[YOUR_SCHOOLOGY_SECRET]
+SCHOOLOGY_OUTPUT_PATH=[YOUR_SCHOOLOGY_OUTPUT_PATH]
```

Comment:
Since you're providing an output directory location, you may want to remove and just output there. Or at least provide it in the example here.  It took me a few tries to get it right, as "output".

Reply:
I will modify the example.